src/Mvc/Mvc.Abstractions/src/Microsoft.AspNetCore.Mvc.Abstractions.WarningSuppressions.xml

```diff
@@ -6,7 +6,12 @@
       <argument>IL2026</argument>
       <property name="Scope">member</property>
       <property name="Target">M:Microsoft.AspNetCore.Mvc.ModelBinding.ModelMetadata.#ctor(Microsoft.AspNetCore.Mvc.ModelBinding.Metadata.ModelMetadataIdentity)</property>
-      <property name="Justification">This warning is left in the product so developers get an ILLink warning when trimming an app when Microsoft.AspNetCore.Mvc.ApiExplorer.IsEnhancedModelMetadataSupported=true.</property>
```

Comment:
I think you want to keep this Justification

Reply:
You're right -- I missed resetting the changes to this file after I regenerated the baselines for the linkability checker.